### ISSUE TITLE
[BUG] - results folder is generation .json and .html results of each of my components, even though my `.achecker.yml` is set to  outputFormat: json

### ISSUE THREAD

**Issue by shroffsmiti** [IBMa/equal-access#213]
## What tool(s) are you using?

- [ ] `accessibility-checker-engine`
- [ ] `accessibility-checker`
- [x] `karma-accessibility-checker`
- [ ] `rule-server`
- [ ] `boilerplates`
- [ ] `other`

## Detailed description

> Node version: 12.4
> Platform: 

> Describe in detail the issue you're having.

## Steps to reproduce the issue

I have my .achecker.yml file somethig like this 
```# required - Provide the authentication token available at http://ibm.biz/A11yToolsLandingPage
# i.e: authToken: 00000000-0000-0000-0000-000000000000/00000000-0000-0000-0000-000000000000

# optional - Specify the rule archive
# i.e. For march rule archive use ruleArchive: 2017MayDeploy
# Default: latest
# Refer to README.md FAQ section below to get the rule archive ID.
ruleArchive : latest

# optional - Specify one or many policies to scan.
# i.e. For one policy use policies: IBM_Accessibility_2017_02
# i.e. Multiple policies: IBM_Accessibility_2017_02,IBM_Accessibility_BETA or refer to below as a list
# Default: null (all policies)
# Refer to README.md FAQ section below to get the policy ID.
policies:
    - IBM_Accessibility
    # - IBM_Accessibility_WCAG21

# optional - Specify one or many violation levels on which to fail the test
#            i.e. If specified violation then the testcase will only fail if
#                 a violation is found during the scan.
# i.e. failLevels: violation
# i.e. failLevels: violation,potential violation or refer to below as a list
# Default: violation, potentialviolation
failLevels:
    - violation
    - potentialviolation

# optional - Specify one or many violation levels which should be reported
#            i.e. If specified violation then in the report it would only contain
#                 results which are level of violation.
# i.e. reportLevels: violation
# i.e. reportLevels: violation,potentialviolation or refer to below as a list
# Default: violation, potentialviolation, recommendation, potentialrecommendation, manual
reportLevels:
    - violation
    - potentialviolation
    - recommendation
    - potentialrecommendation
    - manual

# Optional - Which type should the results be outputted to
#   outputFormat: json
# Default: json
outputFormat: json

# Optional - Specify labels that you would like associated to your scan
#
# i.e.
#   label: Firefox,master,V12,Linux
#   label:
#       - Firefox
#       - master
#       - V12
#       - Linux
# Default: N/A
label:
    - master

# optional - Where the scan results should be saved.
# Default: results
outputFolder: results

# optional - Where the baseline results should be loaded from
# Default: baselines
baselineFolder: test/baseline

# optional - Should Hidden content be scanned
# true --> Yes scan hidden content
# false --> Don't scan hidden content
# Default: false
checkHiddenContent: false```

But still my output file generate .json n .html file for each component.
 
I even change the values on outputFormat to only html still same results. As tried this format as well.
```outputFormat:
- json```

<!-- Provide as much useful information as you can -->
## Additional information
<img width="679" alt="Screen Shot 2020-09-04 at 3 34 43 PM" src="https://user-images.githubusercontent.com/61367807/92282888-334e6d80-eec4-11ea-8660-217abe6dea1c.png">

**Comment by shroffsmiti** [comment]
<img width="679" alt="Screen Shot 2020-09-04 at 3 34 43 PM" src="https://user-images.githubusercontent.com/61367807/92283021-917b5080-eec4-11ea-9637-9e4fc3e57d1b.png">

**Comment by tombrunet** [comment]
Addressed in https://github.com/IBMa/equal-access/pull/227

**Comment by tombrunet** [comment]
@shroffsmiti 3.1.0 was deployed yesterday and should fix this. Please try again.

**Comment by nsharma1989** [comment]
HI, I am still facing this issue with the latest version.

**Comment by tombrunet** [comment]
Still unable to produce using https://github.com/IBMa/equal-access/tree/master/karma-accessibility-checker/boilerplates/basic and the latest version. Please ensure that you have reinstalled the karma checker (clear out node_modules and package-lock and that the package.json version is latest).
